intezer_dynamic.py

```diff
@@ -140,105 +140,233 @@ def get_latest_analysis(self,
                             file_hash: str,
                             private_only: bool = False,
                             **additional_parameters) -> Optional[Dict]:
-        try:
-            return IntezerApi.get_latest_analysis(
-                self=self,
-                file_hash=file_hash,
-                private_only=private_only,
-                additional_parameters=additional_parameters
-            )
-        except HTTPError as e:
-            self.log.debug(
-                f"Unable to get the latest analysis for SHA256 {file_hash} due to '{e}'"
-            )
-            # Occasionally an analysis fails, and HTTPError.GONE is raised
-            if str(HTTPStatus.GONE.value) in repr(e) or HTTPStatus.GONE.name in repr(e):
-                return None
-            else:
-                raise
+        # We will try to connect with the REST API... NO MATTER WHAT
+        while True:
+            try:
+                return IntezerApi.get_latest_analysis(
+                    self=self,
+                    file_hash=file_hash,
+                    private_only=private_only,
+                    additional_parameters=additional_parameters
+                )
+            except ConnectionError as e:
+                self.log.error(
```

Comment:
Having the error logging in the polling loop is probably going to generate a ton of logs for a single failure. Maybe set a flag to only log on the first failed attempt